modules/percolator/src/main/java/org/elasticsearch/percolator/PercolatorFieldMapper.java

```diff
@@ -102,7 +103,7 @@
 
     @Override
     public FieldMapper.Builder getMergeBuilder() {
-        return new Builder(simpleName(), searchExecutionContext, mapUnmappedFieldsAsText).init(this);
+        return new Builder(simpleName(), searchExecutionContext, mapUnmappedFieldsAsText, indexCreatedVersion).init(this);
```

Comment:
I started things out like this, actually, but then felt bad about it because if someone else was to reuse this indexCreatedVersion field in KeywordFieldMapper for other purposes, they might be in for a surprise (not always representing the actual index.version.created). As this only required a small change here in this code, I felt ok with the extra ceremony (that might be useful for other purposes in the future, as I eventually expect every mapper to have this).